systems/subscriber/sim-pool/pb/sim.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+option go_package = "pb/gen";
+package ukama.sim.v1;
```

Comment:
package should be updated to ukama.subscriber.sim.v1

Reply:
I have done the change.